base/linalg/uniformscaling.jl

```diff
@@ -171,15 +171,27 @@ function isapprox(J1::UniformScaling{T}, J2::UniformScaling{S};
             rtol::Real=Base.rtoldefault(T,S), atol::Real=0, nans::Bool=false) where {T<:Number,S<:Number}
     isapprox(J1.λ, J2.λ, rtol=rtol, atol=atol, nans=nans)
 end
-function isapprox(J::UniformScaling,A::AbstractMatrix;kwargs...)
+function isapprox(J::UniformScaling,A::AbstractMatrix;
+                       rtol::Real=rtoldefault(promote_leaf_eltypes(A),eltype(J)),
+                       atol::Real=0, nans::Bool=false, norm::Function=vecnorm)
```

Comment:
This line and the one above it should be deindented such that they line up beneath the `J` after the initial opening paren.